include/z64save.h

```diff
@@ -4,6 +4,28 @@
 #include "ultra64.h"
 #include "z64math.h"
 
+// `_FORCE` means that this request will respond to `forceRisingButtonAlphas`.
+// If set, the buttons will also raise alphas but will also account for disabled buttons
+
+typedef enum {
+    /*  0 */ HUD_VISIBILITY_NO_CHANGE,
+    /*  1 */ HUD_VISIBILITY_NOTHING,
+    /*  2 */ HUD_VISIBILITY_NOTHING_ALT, // Identical to HUD_VISIBILITY_NOTHING
+    /*  3 */ HUD_VISIBILITY_HEARTS_FORCE, // See above
+    /*  4 */ HUD_VISIBILITY_A,
+    /*  5 */ HUD_VISIBILITY_A_HEARTS_MAGIC_FORCE, // See above
+    /*  6 */ HUD_VISIBILITY_A_HEARTS_MAGIC_MINIMAP_FORCE, // See above
+    /*  7 */ HUD_VISIBILITY_ALL_NO_MINIMAP_BY_BTN_STATUS, // Only raises button alphas if not disabled
+    /*  8 */ HUD_VISIBILITY_B,
+    /*  9 */ HUD_VISIBILITY_HEARTS_MAGIC,
+    /* 10 */ HUD_VISIBILITY_B_ALT, // Identical to HUD_VISIBILITY_B
+    /* 11 */ HUD_VISIBILITY_HEARTS,
+    /* 12 */ HUD_VISIBILITY_A_B_MINIMAP,
+    /* 13 */ HUD_VISIBILITY_HEARTS_MAGIC_FORCE, // See above
+    /* 50 */ HUD_VISIBILITY_ALL = 50, // Only raises button alphas if not disabled
```

Comment:
I'm kind of lost with the changes so this has probably been addressed before but, the other `Interface_RaiseButtonAlphas`-using visibility mode `HUD_VISIBILITY_ALL_NO_MINIMAP_BY_BTN_STATUS` has `_BY_BTN_STATUS`, should `HUD_VISIBILITY_ALL` also have that suffix? (`HUD_VISIBILITY_ALL_BY_BTN_STATUS`)

Reply:
Yeah, this is one of those convenience vs consistency things. `HUD_VISIBILITY_ALL_BY_BTN_STATUS` would be consistent because it does use `Interface_RaiseButtonAlphas`, but `HUD_VISIBILITY_ALL` is used quite a bit on the user side while `HUD_VISIBILITY_ALL_BY_BTN_STATUS` is not

Reply:
> HUD_VISIBILITY_ALL is used quite a bit on the user side while HUD_VISIBILITY_ALL_BY_BTN_STATUS is not

:thinking: you mean "`HUD_VISIBILITY_ALL_NO_MINIMAP_BY_BTN_STATUS` is not" (used much) ?